docs-source/spring/content/development/cli/_index.md

```diff
@@ -560,3 +560,179 @@ The following is an example development workflow using the CLI:
          oractl:>config list --service-name myserv --service-profile obaas --service-label 0.1 --property-key ktest2
          400 : "Couldn't find any property for submitted query."
          ```
+8. Use the `GraalVM Compile Commands` to:
+* upload a **.jar** file to the Oracle Backend for Spring Boot and Microservices and its GraalVM compiler service;
+* start a compilation of your microservice to produce an executable native file **.exec**;
+* get last logs available regarding a compilation in progress or terminated;
+* download the **.exec** file to deploy on the backend
+* purge files remaining after a compilation on remote GraalVM compiler service.
+
+The GraalVM Compile Commands are the following:
+
+```cmd
+oractl:>help 
+   
+GraalVM Compile Commands
+       compile-download: Download compiled executable file
+       compile: Compile a service with GraalVM
+       compile-purge: Delete a launched job
+       compile-logs: Compilation progress
+```
+
+   1. Use the `compile` command to upload and automatically start compilation using the following syntax:
+
+```cmd
+   oractl:>help compile
+       NAME
+       compile - Compile a service with GraalVM
+
+       SYNOPSIS
+              compile [--artifact-path String] --help 
+
+       OPTIONS
+              --artifact-path String
+              Service jar to compile location
+              [Mandatory]
+
+              --help or -h 
+              help for compile
+              [Optional]
+```
+
+   Since the compilation of a .jar file via the tool `native-image` does not support cross-compilation, so it must be on the same kind of platform where the application will run, this service guarantees a compilation in the same Operating System and CPU type where the service it will be executed on the Kubernetes cluster.
+
+   The Spring Boot application **pom.xml** with the plugin:
+
+```xml
+<plugin>
+       <groupId>org.graalvm.buildtools</groupId>
+       <artifactId>native-maven-plugin</artifactId>
+</plugin>
+```
+
+The project should be compiled on the developer desktop with GraalVM version 22.3 or later using an **mvn** command. For example:
+  
+```cmd
+  mvn -Pnative native:compile -Dmaven.test.skip=true
+```
+
+   This pre-compilation on your desktop needs to check if there are any issues on the libraries used in your Spring Boot microservice. In addition, your executable jar file must include AOT generated assets such as gnenerated classes and JSON hint files. For additional information, see [Converting Spring Boot Executable Jar] (https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.advanced.converting-executable-jars).
```

Comment:
In the second sentence, change:

executable jar file => executable .jar file

include AOT generated => include ahead-of-time (AOT) generated

gnenerated => generated